packages/shared/src/store/dbHooks.ts

```diff
@@ -227,6 +227,7 @@ export const useChannel = (options: { id: string }) => {
   return useQuery({
     queryKey: [['channel', options]],
     queryFn: () => db.getChannelWithLastPostAndMembers(options),
+    enabled: !!options.id,
```

Comment:
Might as well make the param optional in this case

Reply:
This is an artifact of an earlier implementation on this PR, removing.